### PR TITLE
[Rally] Content description, tabs, and grouping views

### PR DESCRIPTION
**Pull request by JolandaVerhoef**

### CODE REVIEW

File: Rally/app/src/main/java/com/example/compose/rally/ui/overview/OverviewScreen.kt [hunk 1]
```diff
@@ -128,7 +130,9 @@ private fun AlertHeader(onClickSeeAll: () -> Unit) {
 @Composable
 private fun AlertItem(message: String) {
     Row(
-        modifier = Modifier.padding(RallyDefaultPadding),
+        modifier = Modifier
+            .padding(RallyDefaultPadding)
+            .semantics(mergeDescendants = true) {},
```
Comment (JoseAlcerreca) [inline Rally/app/src/main/java/com/example/compose/rally/ui/overview/OverviewScreen.kt:135]: Add a comment explaining, also about the difference with  `Modifier.clearAndSetSemantics {}`

File: Rally/app/src/main/java/com/example/compose/rally/ui/overview/OverviewScreen.kt [hunk 2]
```diff
@@ -138,9 +142,11 @@ private fun AlertItem(message: String) {
         )
         IconButton(
             onClick = {},
-            modifier = Modifier.align(Alignment.Top)
+            modifier = Modifier
+                .align(Alignment.Top)
+                .clearAndSetSemantics {}
         ) {
-            Icon(Icons.Filled.Sort, contentDescription = stringResource(id = R.string.sort))
+            Icon(Icons.Filled.Sort, contentDescription = null)
```
Comment (JoseAlcerreca) [inline Rally/app/src/main/java/com/example/compose/rally/ui/overview/OverviewScreen.kt:149]: why wouldn't you want that?

Comment (JolandaVerhoef) [reply Rally/app/src/main/java/com/example/compose/rally/ui/overview/OverviewScreen.kt:149]: I was trying to figure out what this button would actually do when you press it. Seemed weird to have a "sort" button on each alert. 

Also; having an identical id for a button on each row in a column is considered a bad practice, as voice access users won't be able to identify them, and focusing each of them would be a repeated effort. In this case, I would use a custom action on the row instead, that describes the action the user can take.

File: Rally/app/src/main/java/com/example/compose/rally/ui/overview/OverviewScreen.kt (hunk 1 above)
Comment (JolandaVerhoef) [reply Rally/app/src/main/java/com/example/compose/rally/ui/overview/OverviewScreen.kt:135]: There's barely any comments in this sample as it is. I'm adding these same properties to all sample apps. Is it really desired to add a comment "Merges the semantics properties of the descendants" to each of these? Or would it be best to have good documentation on these topics and leave it like this.

Comment (JoseAlcerreca) [reply Rally/app/src/main/java/com/example/compose/rally/ui/overview/OverviewScreen.kt:135]: The comment shouldn't say what it does but why it does it in this case.

> There's barely any comments in this sample as it is. 
Not an excuse :)

**Review by JoseAlcerreca — APPROVED** [review]

**Pull request merged**